### ISSUE TITLE
ep-async-api-importer: Schema Name and Version are ignored from extension

### ISSUE THREAD

**Issue by TamimiGitHub** [SolaceLabs/solace-tools-typescript#16]
## Issue
The importer does not check `x-ep-schema-name` and `x-ep-schema-version` in the schema. It overrides the schema name and version to the name/version of the event.

## Importer Version
0.52.0

Attached is a sample AsyncAPI spec file
[pos-system.yaml.zip](https://github.com/SolaceLabs/solace-tools-typescript/files/10457890/pos-system.yaml.zip)

**Comment by ricardojosegomezulmke** [comment]
neither schema name nor schema version extensions are part of the importer / async document.
Best approach:
implement all relevant extensions event portal uses:
- ep event name (need to consider inadequate ep extension placement as well. it needs to go into the channel/message association instead of the message to support multiple messages per channel AND re-using same message across channels)
- schema name

the versions could be used as a 'suggestion' during import as well.

**Comment by TamimiGitHub** [comment]
Schema name and Schema version extensions are currently part of the async document exported from EP. So for example the `Product` schema has the ep extensions for name and version ` x-ep-schema-version` and `x-ep-schema-name`
```
components:
  schemas:
    Product:
      x-ep-schema-version: "0.1.0"
      x-ep-schema-state-id: "2"
      x-ep-schema-version-id: "cffhtgg3llq"
      $schema: "http://json-schema.org/draft-06/schema#"
      x-ep-schema-id: "80b8ejji5f3"
      x-ep-schema-state-name: "RELEASED"
      x-ep-schema-name: "Product"
      type: "object"
      properties:
        quantity:
          type: "integer"
        price:
          type: "number"
        description:
          type: "string"
        currency:
          type: "string"
        sku:
          type: "string"
      required:
      - "description"
      - "sku"
      - "quantity"
      - "price"
      - "currency"
      x-ep-schema-version-displayname: ""
```

So at this point we can add a feature to the importer to use these extensions when importing the spec to EP

**Comment by TamimiGitHub** [comment]
Fixed in 169053fff59b635b40229d5c94044cbb90450a7a
